src/frisby/expects.js

```diff
@@ -19,6 +19,16 @@ function incrementAssertionCount() {
 
 const expects = {
 
+  responseTime(response, maxResponseTime) {
+    incrementAssertionCount();
+
+    assert.ok(
+      maxResponseTime >= response.responseTime,
+      `Request took longer than ${maxResponseTime}ms: (${response.responseTime}ms).`
+    );
+  },
+
```

Comment:
Please remove this line.

Reply:
Not 100% sure if you mean the empty line (30) or the line break. I guessed and removed the empty line.